.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["@babel/preset-env"],
+  "presets": ["@babel/preset-env", "@babel/preset-typescript"],
```

Comment:
Is this required? Isn't this going to clash with `tsc`?

Reply:
I thought it was required and produced no side effects, but… I just tried running the tests without it again and everything is fine.